pkg/eventhandler/eventhandlers.go

```diff
@@ -135,6 +138,21 @@ func (eh *EventHandler) startK8sJob(k8s k8sutils.K8s, action *config.Action, jso
 		start: time.Now(),
 	}
 
+	// To execute all tasks atomically, we check all images
+	// before we start executing a single task of a job
+	for _, task := range action.Tasks {
+		if !eh.AllowedImages.Contains(task.Image) {
+			log.Printf("Image %s is not in the allowed!\n", task.Image)
```

Comment:
I would change this message to `"Forbidden: Image %s is not part of allowedImageList.\n"`